server/src/endpoints/obelisk/startObservingObelisk.ts

```diff
@@ -0,0 +1,24 @@
+import DB from '../../redis'
+import { User } from '../../user'
+import { AuthenticatedEndpointFunction, LogFn } from '../../endpoint'
+
+const startObservingObelisk: AuthenticatedEndpointFunction = async (user: User, inputs: any, log: LogFn) => {
+  // The DB just knows there's a single obelisk, and doesn't distinguish between sidebar and the room
+  const notes = await DB.getRoomNotes('obelisk')
+
+  return {
+    groupManagementTasks: [
```

Comment:
OK SO we have a bug in authenticatedAzureWrap, see 

https://github.com/Roguelike-Celebration/azure-mud/blob/fe8047ad8093a79d5e2e2a66be0294dd1f5ac79c/server/src/azureWrap.ts#L48-L49

https://github.com/Roguelike-Celebration/azure-mud/blob/fe8047ad8093a79d5e2e2a66be0294dd1f5ac79c/server/src/azureWrap.ts#L75-L76

The bug is that if you have no messages groupManagementTasks 500s because it's trying to push onto an array that's only created if we have messages, previously this must have been a non-issue because we only used this when we were notifying users anyways but it's an issue now, so either the bug needs to be fixed or `messages: []` needs to get added.

Reply:
fixed in latest commit